### DISCUSSION TITLE
YAML front matter in pull request templates

### DISCUSSION THREAD

**Discussion by LorenzHenk** [community/community#4527] [Pull Requests]
For [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository) it's possible to define default values via front matter.

This feature would be highly appreciated for PR templates :slightly_smiling_face: 

Related: https://github.com/isaacs/github/issues/1252

**Comment by Torxed** [comment]
No traction or answers, so I created a separate request with a different title more similar to one that did get traction:
 * Mine: https://github.com/orgs/community/discussions/27897
 * Popular one: https://github.com/orgs/community/discussions/2838
